README.md

```diff
@@ -6,7 +6,7 @@ This extension implements the most important feature of MailChimp: Let the users
 
 ## Requirements
 
-- TYPO3 CMS 8.7 - 9.5
+- TYPO3 CMS 8.7 - 10.0
```

Comment:
Is it really still v8.7 compatible?

Reply:
I did not test, but we decided that the minimal version should be forced to 9. So the requirements should be adjusted for this version.

Reply:
Fixed